avocado-setup.py

```diff
@@ -367,8 +370,12 @@ def run_test(testsuite, avocado_bin):
         testsuite.runstatus("Not_Run", "Command execution failed")
         return
     logger.info('')
-    result_link = "%s/job.log" % testsuite.jobdir()
-    testsuite.runstatus("Run", "Successfully executed", result_link)
+    result_link = testsuite.jobdir()
+    if result_link:
+        result_link += "/job.log"
+        testsuite.runstatus("Run", "Successfully executed", result_link)
+    else:
+        testsuite.runstatus("Not_Run", "Finding job.log failed")
```

Comment:
`Unable to find job log file` or `missing job log file`

Reply:
@sathnaga thanks, pushed.